kvk/Issues/official/description.xml

```diff
@@ -456,7 +456,7 @@
 			</Name>
 			<Description>
 				<en>Indication of whether someone is authorized</en>
-				<nl>Indicatie of iemand bevoegd is</nl>
+				<nl>Indicatie of iemand bevoegd is ('Ja', 'Nee' of 'Niet vastgesteld')</nl>
```

Comment:
Maybe as a nice-to-know, we have a displayHint="yesno" to make sure these values are actually translated correctly in the English translation of the app. This might be useful to you.
https://irma.app/docs/schemes/

Reply:
the value of this field starts with a capital, will this work; ```displayHint="JaNee"```

Reply:
Here you can see how the `displayHint="yesno"` works. https://github.com/privacybydesign/irmamobile/blob/17eafc899c70b42b11f3a37f920a249125920d7a/lib/src/models/attribute_value.dart#L92